experimental/common-expression-language/README.md

```diff
@@ -1,33 +1,41 @@
-# Common Expression Language ***_[PREVIEW]_***
+# Adaptive expressions ***_[PREVIEW]_***
 
-> See [here](#Change-Log) for what's new in 4.6 PREVIEW release.
+Bots use expressions to evaluate the outcome of a condition based on runtime information available in memory to the dialog or the [Language Generation](../language-generation) system. These evaluations determine how your bot reacts to user input and other factors that impact bot functionality.
 
-Bots, like any other application, require use of expressions to evaluate outcome of a condition based on runtime information available in memory or to the dialog or the language generation system. 
+Adaptive expressions were created to address this core need as well as provide an adaptive expression language that can used with the Bot Framework SDK and other conversational AI components, like [Bot Framework Composer](https://github.com/microsoft/BotFramework-Composer#microsoft-bot-framework-composer-preview), [Language Generation](../language-generation), [Adaptive dialogs](../adaptive-dialog), and [Adaptive Cards](https://docs.microsoft.com/adaptive-cards/).
 
-Common expression language was put together to address this core need as well as to rationalize and snap to a common expression language that will be used across Bot Framework SDK and other conversational AI components that need an expression language.
+An adaptive expression can contain one or more [operators](#Operators), [variables](#Variables), [explicit values](#Explicit-values), [pre-built functions](./prebuilt-functions.md) or [Language Generation templates](../language-generation/docs/lg-file-format.md#templates).
```

Comment:
I think this should be re-worded to `explicit values, functions ([pre-built](./prebuilt-functions.md) or [custom](./extend-functions.md)`.  

Consumers of AdaptiveExpressions also have the capability to inject additional supported functions. As an example, with [Language generation](../language-generation/README.md), all [templates](../language-generation/docs/lg-file-format.md#templates) are available as functions as well as [additional functions](../language-generation/docs/Functions-injected-from-LG.md) that are only available within that components use of AdaptiveExpressions.